test/test_transformers.py

```diff
@@ -1843,6 +1839,10 @@ def test_flash_attention_vs_math_ref_grads(self, batch_size: int, seq_len_q: int
 
         upstream_grad = torch.rand_like(out, requires_grad=False)
 
+        # backward for flash attention on sm86 and sm89 for headdim > 64 currently disabled
```

Comment:
hmmm this isn't quite right, we want the forward to fail not the backward

Reply:
unless the inputs don't require _grad then the forward can succede but this is up for debate as well

Reply:
Followed up offline. The highlighted test calls the private function directly so this is fine; the sdpa dispatcher test where we want it to fail in the forward is here: https://github.com/pytorch/pytorch/blob/97afbcbc8007857a51c85e9c61fe6d80564ef1f9/test/test_transformers.py#L1071